src/features/cpu_load/updater.rs

```diff
@@ -6,7 +6,12 @@ use crate::wrapper::file;
 use super::Data;
 use super::FEATURE_NAME;
 
+use regex::Regex;
+
 const PATH_LOADAVG: &str = "/proc/loadavg";
+const PATH_NPROC: &str = "/proc/cpuinfo";
+
+const NPROC_REGEX: &str = r"processor\s+: \d+";
```

Comment:
To prevent false positive matches:
```suggestion
const NPROC_REGEX: &str = r"^processor\s+: \d+$";
```